Formula/kubectx.rb

```diff
@@ -0,0 +1,28 @@
+class Kubectx < Formula
+  desc "Tool that can switch between kubectl contexts easily and create aliases"
+  homepage "https://github.com/ahmetb/kubectx"
+  url "https://github.com/ahmetb/kubectx/archive/v0.3.1.tar.gz"
+  sha256 "4e995f5bec6f41c8d5b6e77f413a58ead077816348e72de26dde3655ec2b7d0b"
+  head "https://github.com/ahmetb/kubectx.git", :branch => "master"
+  bottle :unneeded
+
+  option "with-short-names", "link as \"kctx\" and \"kns\" instead"
+
+  depends_on "kubernetes-cli" => :run
+
+  def install
+    bin.install "kubectx" => build.with?("short-names") ? "kctx" : "kubectx"
+    bin.install "kubens" => build.with?("short-names") ? "kns" : "kubens"
+    include.install "utils.bash"
+
+    bash_completion.install "completion/kubectx.bash" => "kubectx"
+    bash_completion.install "completion/kubens.bash" => "kubens"
+    zsh_completion.install "completion/kubectx.zsh" => "_kubectx"
+    zsh_completion.install "completion/kubens.zsh" => "_kubens"
+  end
+
+  test do
+    assert_match "USAGE:", shell_output("#{bin}/kubectx -h 2>&1")
```

Comment:
Per the [docs](http://www.rubydoc.info/github/Homebrew/brew/Homebrew%2FAssertions%3Ashell_output), you can pass a second argument to `shell_output` with the expected exit code. If it's 1, then it would look something like: 
```
shell_output("#{bin}/kubectx -h 2>&1",1)
```